app/assets/javascripts/users.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
+# You can use CoffeeScript in this file: http://coffeescript.org/
```

Comment:
do we need to keep these coffee files? Is anyone using them>?

Reply:
I think it should be safe to get rid of them. They all should be auto-generated, and we'd need to replace them with .js files to do anything with them anyways